src/components/movieDetails/MovieDetails.tsx

```diff
@@ -112,7 +113,7 @@ const MovieDetails = () => {
   const {backdrop_path} = movieDetails;
 
   if (isLoading) {
-    return <Loader />;
+    return <Loader size={'large'} color={colors.primary} />;
```

Comment:
set it as default so that you don't have to pass props everytime if you are using same size and color